README.md

```diff
@@ -139,6 +139,11 @@ jobs:
 - **Description**: The directory containing compilation database (like compile_commands.json) file.
 - Default: ''
 
+#### `extra-arg`
```

Comment:
1. This should be a plural form because there's no way to specify this multiple times like clang-tidy and cpp-linter CLI does.
2. We need to discourage the use of spaces unless users are specifying multiple args.